### DISCUSSION TITLE
cant download latex

### DISCUSSION THREAD

**Discussion by YavorJJ** [Cyfrin/security-and-auditing-full-course-s23#214] [Q&A]
I have been trying to run this command: sudo apt-get install texlive-full. 
when I ran it, it actually started downloading, but my mistake was that I killed the terminal.
now when I try to run it again: this is the output the terminal is giving me: 
![image](https://github.com/user-attachments/assets/e67f8045-eacf-4257-b275-990a6437b3c2)

am I doomed here because of my stupid mistake I did? if someone had this output and solved it, I would love for some help!

**Comment by EngrPips** [comment]
This is a tough one. I've never encountered this before, but can't you uninstall and re-install the package?

**Reply by YavorJJ** [reply]
> This is a tough one. I've never encountered this before, but can't you uninstall and re-install the package?

I saw a comment mins ago about re-install and install, but still the terminal gives me the same output. and yes you are right, its indeed a tough one, I still can't see someone encountring this problem, but I will still continue looking around
 to see if there is any fix.

**Reply by YavorJJ** [reply]
so, I saw this website: https://wiki.debian.org/Teams/Dpkg/FAQ#Q:_What_can_be_done_when_the_dpkg_lock_is_held.3F
I am trying this for myself now to see if I can get out of this situation. I will update you. I am pasting the link for this website here, for someone that may struggle with this.

**Reply by EngrPips** [reply]
I hope this fixes it for you. I am looking forward to hearing from you.

**Reply by YavorJJ** [reply]
> I hope this fixes it for you. I am looking forward to hearing from you.

it did not fix, I read so much about this, trying to figure it out, kept having problems with installation, while the system and terminal was not working and stuff going on was hard. However I kept reasearching, and finaly with diffrent tools I managed to create my pdf :). Stuff took me a day and something..

Its good that I am finally back, and let me say for the people with this lock problem that I had, the best thing is to ask Chatgpt and always research.

**Reply by EngrPips** [reply]
Indeed, you are now becoming prominent in solving your own problem. That is what it means to be a programmer `figure things out`
